operations/network/customer_accessible_networks/can_to_chn/network/network_upgrade_1.2_to_1.3.md

```diff
@@ -1,16 +1,17 @@
 # Management Network Upgrade CSM 1.2 to 1.3
 
-- [Management network changelog](#management-network-changelog)
-- [When to perform switch upgrades](#when-to-perform-switch-upgrades)
-- [Prerequisites](#prerequisites)
-- [Upgrade CANU to the latest version](#upgrade-canu-to-the-latest-version)
-- [Backup running switch configurations](#backup-running-switch-configurations)
-- [Retrieve SLS data](#retrieve-sls-data)
-- [Generate CSM 1.3 switch configurations](#generate-csm-13-switch-configurations)
-- [Discover CSM 1.2 to CSM 1.3 configuration changes](#discover-csm-12-to-csm-13-configuration-changes)
-- [Test the network prior to update](#test-the-network-prior-to-update)
-- [Apply and save the new switch configurations](#apply-and-save-the-new-switch-configurations)
-- [Test the network after update](#test-the-network-after-update)
+- [Management Network Upgrade CSM 1.2 to 1.3](#management-network-upgrade-csm-12-to-13)
+  - [Management network changelog](#management-network-changelog)
+  - [When to perform switch upgrades](#when-to-perform-switch-upgrades)
+  - [Prerequisites](#prerequisites)
+  - [Upgrade CANU to the latest version](#upgrade-canu-to-the-latest-version)
+  - [Backup running switch configurations](#backup-running-switch-configurations)
+  - [Retrieve SLS data](#retrieve-sls-data)
+  - [Generate CSM 1.3 switch configurations](#generate-csm-13-switch-configurations)
+  - [Discover CSM 1.2 to CSM 1.3 configuration changes](#discover-csm-12-to-csm-13-configuration-changes)
+  - [Test the network prior to update](#test-the-network-prior-to-update)
+  - [Apply and save the new switch configurations](#apply-and-save-the-new-switch-configurations)
+  - [Test the network after update](#test-the-network-after-update)
```

Comment:
```suggestion
- [Management network changelog](#management-network-changelog)
- [When to perform switch upgrades](#when-to-perform-switch-upgrades)
- [Prerequisites](#prerequisites)
- [Upgrade CANU to the latest version](#upgrade-canu-to-the-latest-version)
- [Backup running switch configurations](#backup-running-switch-configurations)
- [Retrieve SLS data](#retrieve-sls-data)
- [Generate CSM 1.3 switch configurations](#generate-csm-13-switch-configurations)
- [Discover CSM 1.2 to CSM 1.3 configuration changes](#discover-csm-12-to-csm-13-configuration-changes)
- [Test the network prior to update](#test-the-network-prior-to-update)
- [Apply and save the new switch configurations](#apply-and-save-the-new-switch-configurations)
- [Test the network after update](#test-the-network-after-update)
```

Reply:
We avoid including the page itself in its own TOC, since it does not add any information, just clutter. I know that some editors put it in there, but that is mainly for cases where a document has multiple top-level headers. However, in this repository we do not allow multiple top-level headers, so for this repository it should be omitted.

Reply:
I no longer see the issue. My bad. Marking this resolved.